storage/ndb/rest-server/rest-api-server/internal/handlers/feature_store/handler.go

```diff
@@ -532,6 +544,10 @@ func GetBatchPkReadParams(metadata *feature_store.FeatureViewMetadata, entries *
 		testDb := fgFeature.FeatureStoreName
 		testTable := fgFeature.TableName
 
+		if fgFeature.IsSpine() {
```

Comment:
I think it should still work if you dont skip? because when the table is not found, it will return 404 anyway.